synapseclient/core/multithread_download/download_threads.py

```diff
@@ -186,6 +186,29 @@ def _pre_signed_url_expiration_time(url: str) -> datetime:
     :return: datetime in UTC of when the url will expire
     """
     parsed_query: dict = parse_qs(urlparse(url).query)
+    if (
```

Comment:
NIT: I don't think you need any of the `is not None`'s in this function - the same outcome can be achieved with 
```
    if (
        parsed_query.get("X-Amz-Date")
        and parsed_query.get("X-Amz-Expires")
    ):
```
for example.